src/index.js

```diff
@@ -27,9 +29,9 @@ export default class toColor {
   getColor(count = 0) {
     const h = this._pickHue();
     const s = this._pickSaturation(h);
```

Comment:
We're ignoring `h` now, do we still want to pass it in?

```suggestion
    const s = this._pickSaturation();
```